hubapi/README.MD

```diff
@@ -0,0 +1,75 @@
+`HubAPI`
+The HubAPI supports operations for listing jina hub images
```

Comment:
When we're talking about Jina Hub outside of the actual command line, call it **J**ina **H**ub

On command line or code blocks use `jina hub`